src/main/java/org/trigon/hardware/BaseInputs.java

```diff
@@ -37,6 +48,13 @@ public double getSignal(String signalName) {
         return value.getDouble();
     }
 
+    /**
+     * Gets a threaded signal.
+     * Threaded signals use threading to process certain signals separately. Should be used for odometry and other fast signals.
```

Comment:
Threaded signals use threading to process certain signals separately at a faster rate